### ISSUE TITLE
PopupControl is rendered misplaced and with surrounding black screen

### ISSUE THREAD

**Issue by jperedadnr** [gluonhq/substrate#534]
Controls like ComboBox, DatePicker, ColorPicker, Menu, use a PopupControl to render a popup right below them. 
On Android, the popup is rendered in the left corner of the screen, with a black screen around. However, tapping at the "real" position, still works to select items from the control.

**Comment by DeanWookey** [comment]
@jperedadnr is there a way to debug issues like these? Can you point me in the right direction and maybe explain how you guys are working?

Right now I'm thinking of building the java source with print statements in relevant places and trying to get it to use those javafx libraries. Before I embark on that, maybe you could let me know if there is a better way, or any tips?

**Comment by jperedadnr** [comment]
This issue is also related to #557 

@DeanWookey It is hard to test on Android, as it requires doing an static build of OpenJFX for Android and applying some required configuration for Substrate.

**Comment by molbertz** [comment]
@jperedadnr I just came across this issue myself. Is there any progress for a fix or a workaround?

**Comment by jperedadnr** [comment]
Fixed with #554 

Until a new release comes out, It can be tested with:

```
<plugin>
    <groupId>com.gluonhq</groupId>
    <artifactId>client-maven-plugin</artifactId>
    <version>0.1.39-SNAPSHOT</version>
   <configuration>
        <target>${clientTarget}</target>
        <mainClass>${mainClassName}</mainClass>
         ...
   </configuration>
</plugin>

<pluginRepositories>
    <pluginRepository>
        <id>Sonatype</id>
        <url>https://oss.sonatype.org/content/repositories/snapshots/</url>
    </pluginRepository>
</pluginRepositories>
```
